puppet/zulip_ops/manifests/base.pp

```diff
@@ -36,6 +34,12 @@
                          ]
   package { $org_base_packages: ensure => "installed" }
 
+  exec {"pip_ec2_wal-e":
+    # pip dependencies to run ec2-configure-interfaces and wal-e
+    command  => "/usr/bin/pip2 install 'boto==2.4.0' 'netifaces==0.10.6'",
+    require  => Package['python-pip'],
+  }
+
```

Comment:
Maybe I'm missing something, but I don't see any benefit to this change.  It's better to get dependencies via `apt` than `pip`, in general; we only do something different for `wal-e` because there isn't a version available in apt...

Reply:
The added benefit is the version is pinned, and a possibility to migrate to the python3 version of wal-e.